### DISCUSSION TITLE
Handle Camera, LightingSource, SoundSource and other SceneConditon Visibility Items

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#211] [Feature Request]
This kind of goes along with Exposing AddConnectionToScenes and RemoveConnectionToScenes.

Currently as far as I can tell, only Renderers are managed by Scene Condition and Visibility. If I want an entire Scene to not be Visible to a client but kept loaded,  Main Camera, Lighting, Sound  and any other similar visual or audio components a client can see or hear, should be handled as well. 

Currently, It seems the Host Client is affected the most by this as they will be in the situation where they are not an observer of the servers scenes, but if functionality were to be added to also load scenes on clients, and remove the clients from their loaded scene then this will affect  them as well.

Currently in a game I am utilizing this where host is in a scene but not observing, and I have to build custom scripts to handle the lighting and main camera in those scenes, unless I am missing something.

**Comment by FirstGearGames** [comment]
Per our discussion, this is being declined. It's way too much overhead and the list would be never ending. We can discuss a way to let you hook into when renders are hidden/shown for clientHost for your custom logic though. You know where to find me!
